### ISSUE TITLE
Support Reporting Lost Items

### ISSUE THREAD

**Issue by mmackay03** [zgordon01/LostAndFound#1]
Allow for users to enter in an item that has been reported lost and the owners information.

**Comment by zgordon01** [comment]
Do you mean that they can add a record when someone calls in, reporting that something has been lost? So they do not necessarily have the item turned in, but they know it is lost somewhere.

**Comment by mmackay03** [comment]
Yes, that is exactly what I mean. They currently do this with their excel doc.
